### PR TITLE
Use repair step instead of install.php and update.php

### PR DESCRIPTION
**Pull request by MorrisJobke**

### CODE REVIEW

**Comment by MorrisJobke** [comment]
@nickvergessen Any idea why the NC 20 app code check complains about this:

```
Invalid appinfo.xml file found: Element 'repair-steps': This element is not expected. Expected is one of ( activity, dashboard, fulltextsearch, navigations, contactsmenu, collaboration, sabre, public ).
465
466App is not compliant
```

The info.xsd properly contains the `repair-steps` element. :/

**Comment by nickvergessen** [comment]
The php checker unluckily cares about the order.

File: appinfo/info.xml [hunk 1]
```diff
@@ -24,4 +24,9 @@
         <admin>OCA\Survey_Client\Settings\AdminSettings</admin>
         <admin-section>OCA\Survey_Client\Settings\AdminSection</admin-section>
     </settings>
+	<repair-steps>
```
Comment (ChristophWurst) [inline appinfo/info.xml:27]: indentation is off

File: lib/Migration/SendAdminNotification.php [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+	public function __construct(IJobList $jobList) {
+		$this->jobList = $jobList;
+	}
+
+	public function getName(): string {
+		return 'Send an admin notification if monthly report is disabled';
+	}
+
+	public function run(IOutput $output): void {
+		if (!$this->jobList->has('OCA\Survey_Client\BackgroundJobs\MonthlyReport', null)) {
```
Comment (ChristophWurst) [inline lib/Migration/SendAdminNotification.php:46]: couldn't you use `::class`?

Comment (MorrisJobke) [reply lib/Migration/SendAdminNotification.php:46]: I wanted it to be exactly the same as before. I'm not sure if a leading backslash is added in the `::class` case.

Comment (ChristophWurst) [reply lib/Migration/SendAdminNotification.php:46]: Nope, there is none
Bildschirmfoto von 2020-07-14 11-01-16

Comment (nickvergessen) [reply lib/Migration/SendAdminNotification.php:46]: Yeah the code is from php5 times where `::class` didnt work, it was only added in php 5.5

File: appinfo/info.xml (hunk 1 above)
Comment (nickvergessen) [reply appinfo/info.xml]: actually the rest is of, its tabs vs. spaces

**Pull request merged**